report/src/system_evaluation/phase_one_evalation.tex

```diff
@@ -0,0 +1,23 @@
+\section{Phase One: Map Creation}\label{sec:eval_phase1}
+To collect measurements for the creation of the map, a person holding a Samsung Galaxy A53 5G phone, with the application described in Section \ref{sec:data_collection_implementation} installed and running, traverses systematically through the room and stops at each collection point to collect measurements for the map. 
```

Comment:
```suggestion
A person equipped with a Samsung Galaxy A53 5G phone, running the application described in Section \ref{sec:data_collection_implementation}, systematically traverses the room. 
At each collection point, they stop to gather measurements for the map creation.
```

Reply:
Very long sentence - to show you the power of short sentences, I sawed this sentence in half.